### PR TITLE
Gpload add case

### PR DESCRIPTION
**Pull request by xiaoxiaoHe-E**
1. separate gplaod test into different files
  TEST_local_legacy.py,  includes the old test cases
  TEST_local_base.py, includes the basic function of gpload test
  TEST_local_<feature>.py, includes test cases of different features
2. add  gpload test cases about command config

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/bin/gpload_test/.gitignore [hunk 1]
```diff
@@ -4,9 +4,12 @@ gpload/config
 gpload2/config
 gpload2/setup.out
 gpload2/data_file.txt
+gpload2/data_file2.txt
+gpload2/data_file.csv
+gpload2/data/large_file.csv
+gpload2/data_file*
```
Comment (avamingli) [inline gpMgmt/bin/gpload_test/.gitignore:10]: `data_file*` has already included the file above

File: gpMgmt/bin/gpload_test/gpload2/TEST_local_config.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    copy_data('external_file_01.txt','data_file.txt')
+    write_config_file(local_host=['127.0.0.1'],format='text',file='data_file.txt',table='texttable')
+    write_config_file(config='config/config_file2',local_host=None,format='text',file='data_file.txt',table='texttable')
+    write_config_file(config='config/config_file3',local_host=['123.123.1.1'],format='text',file='data_file.txt',table='texttable')
+    f = open('query60.sql','w')
+    f.write("\\! gpload -f "+mkpath('config/config_file')+"\n")
+    f.write("\\! gpload -f "+mkpath('config/config_file2')+"\n")
+    f.write("\\! gpload -f "+mkpath('config/config_file3')+"\n")
+    f.close()
+'''
```
Comment (avamingli) [inline gpMgmt/bin/gpload_test/gpload2/TEST_local_config.py:120]: Does this case has been fixed?

Comment (xiaoxiaoHe-E) [reply gpMgmt/bin/gpload_test/gpload2/TEST_local_config.py:120]: No, we don't need to fix any gpload now.

File: gpMgmt/bin/gpload_test/gpload2/TEST_local_legacy.py [hunk 3]
```diff
@@ -0,0 +1,406 @@
+from TEST_local_base import *
+
+def test_00_gpload_formatOpts_setup():
```
Comment (huiliang-liu) [inline gpMgmt/bin/gpload_test/gpload2/TEST_local_legacy.py:3]: it is duplicated with TEST_local_base.py, we should remove one of them.

Comment (xiaoxiaoHe-E) [reply gpMgmt/bin/gpload_test/gpload2/TEST_local_legacy.py:3]: OK

**Comment by huiliang-liu** [comment]
seems there are more ans files than case? for example ans 64 and later

**Pull request merged**